Assignment/solution_student/css/style.css

```diff
@@ -34,8 +36,251 @@ body {
   color: black;
   font-size: 16px;
   background: white;
+  margin: 0 auto;
+  width: 70%;
+  /*    line-height: 1.5; //// I don't think I need this */
+  clear: both;
+  font-family: 'Open Sans', sans-serif;
+  text-decoration: none;
+}
+
+nav {
+
+	display: table;
+    height: 87px;
+    width: 100%;
+
+}
+
+.container {
+	float: left;
+	margin: 0 auto;
+}
+
+nav ul li {
+
+    height: 87px;
+    width: 15%;  /* (100 / numItems)% */
+    line-height: 87px;
+    text-align: center;
+    white-space: nowrap;
+    font-size: 16px;
+
+}
+
+.home-link {
+	width: 20%;
+	margin: 0 auto;
+	margin-right: 275px;
+	margin-left: 30px;
+	float: left;
+	font-weight: 800;
+	font-size: 35px;
+	color: white;
+	text-align: center;
+	padding-top: 30px;
+	padding-left: 15px;
+}
+
+.home-link a {
+	text-decoration: none;
+	color: white;
+
+}
+
+.main-header {
+
+	background-color: #033048;
+}
+
+
+.nav-container a {
+	text-decoration: none;
+	color: #f9e42e;
+	font-weight: 600;
+
+}
+
+nav ul li a:hover {
+	color: white;	
+	background-color: #f9e42e;
+	border-radius: 5px; 
+	transform: scale(1.0);
+	transition: all .150s ease-in-out;
+	padding: 5px;
+}
+
+.main-header nav ul {
+	list-style: none;
+}
+
+.main-header nav ul li {
+	display: inline-block;
+}
+
+
+.main-content {
+	
+	background-color: #606161;
+
+}
+
+.first-title {
+	margin: 0 auto;
+}
+
+
+nav {
+
+	padding: 1px auto;
+}
+
+.nav-container {
+	width: 100%;
+}
+
+.nav-container li {
+	width: 100px;
+	padding-right: 35px;
+}
+
+.main-content {
+	background-color: #F2F6F5;
+	padding-top: 85px;
+	padding-right: 280px;
+	padding-left: 90px;
+	width: 100%;
+
+}
+
+.first-title, .second-title {
+	padding-bottom: 25px;
+	color: #001E49;
+	font-weight: 600;
+	font-size: 40px;
 }
 
+.top-content-photo {
+	border-radius: 5px;
+}
+
+.main-content-paragraphs {
+
+	font-weight: 300;
+	font-size: 24px;
+}
+
+.read-more {
+
+	font-size: 24px;
+	font-weight: 600;
+	text-decoration: none;
+
+}
+
+.read-more-positioning {
+	padding-bottom: 30px;
+	padding-left: 80%;
+}
+
+.space {
+	padding-bottom: 86px;
+	background-color: whitesmoke;
+	margin-top: -8px;
+}
+
+.signup-container {
+	text-align: center;
+	background-color: yellow;
+	padding: 1%;
+}
+
+button {
+	background-color: #033048;
+	padding: 1%;
+	color: white;
+	font-family: 'Open Sans', sans-serif;
```

Comment:
This isn't needed, since you've declared the Open Sans font on the body, everything inherits it